### ISSUE TITLE
missing dependencies

### ISSUE THREAD

**Issue by ibaca** [niloc132/worker-db#1]
To play with this project you need to upload the websokets project and `mn_dot_traffic_3m.csv` 😆

**Comment by niloc132** [comment]
Yes, and you also need my custom build of rpc-over-websockets and rpc-across-workers...

I'll set the link up for downloads, otherwise poke the demo?

(How did you find this? Are you stalking me?)

**Comment by niloc132** [comment]
https://colinalworth.com/mn_dot_traffic_3m.csv
https://colinalworth.com/mn_dot_traffic_6m.csv

**Comment by ibaca** [comment]
I think I'm going to refactor this project and remove the web-sockets and RPC tools, not bc is not awesome and might be pretty useful for the future but bc those extra tools make it too much awesome (actually complexity, too many new experimental things) and hehe right now I'm just interested in the web-worker inter-communication thing 👍.

**Comment by niloc132** [comment]
Yeah for sure - the next step here is a live scrolling grid, paging items from the worker. The server already pushes items to the client if the query requests them (try something like `equal("STA", "223")` in the top search query and watch the console as new results stream in), and get them to keep updating the worker store, and possibly push to the UI if needed (or at least show a "updates available, please query again").

Also, I put the code up so other people can take advantage of the CQEngine emulation.

If you just want the worker serialization, try the upstream jar, you shouldnt need this at all. There is a simple sample project there, doing simple calculation in the worker.
